### ISSUE TITLE
Sorting options are marked up as radio buttons

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#38]
As a sighted keyboard user, I want to use the sorting options at the top of the product grid. However, they are confusing to me because I can't tab through them all with my keyboard, I can only tab to the first one.

The reason why is that they are marked up as radio buttons with the radio button itself hidden. Radio buttons work by default with the left and right arrow keys but I wouldn't know they are radio buttons because they are hidden.

Solution: Mark these up as buttons. Buttons should be used for interactions that keep the user on the same page.
